### PR TITLE
added flip behaviour

### PR DESCRIPTION
**Pull request by vredchenko**
Hi guys, please review!

### CODE REVIEW

**Comment by Kshavlovsky** [comment]
Looks pretty!

**Comment by KolianN** [comment]
its very nice thanks you very much!!!!

**Comment by MollyLarchik** [comment]
hi)))!!!!!!!

**Comment by Liza2003** [comment]
Looks good.I so happy because I participated

File: index.html [hunk 1]
```diff
@@ -10,18 +10,25 @@
...
-      </article>
+    <section class="flip-container" ontouchstart="this.classList.toggle('hover');">
+
+      <div class="flipper">
+
+        <article class="front">
+          <h2>Semantics</h2>
+          <p>[s'emanti]</p>
+          <img src="" />
+        </article>
```
Comment (vredchenko) [inline index.html]: what is this?

File: index.html [hunk 2]
```diff
@@ -10,18 +10,25 @@
   <body>
     <h1>Vocabulary</h1>
```
Comment (KolianN) [inline index.html]: what is this

File: index.html (hunk 1 above)
Comment (Liza2003) [inline index.html]: Why you did this

**Comment by MarkoFeeD** [comment]
Great

**Pull request merged**